### DISCUSSION TITLE
Does having Embeding from other sites on my Github.io page count towards the Bandwidth?

### DISCUSSION THREAD

**Discussion by Senseal** [community/community#23921] [Other Feature Feedback, Questions, & Ideas]
<p>I was wondering when an embed that grabs a video/image from another webpage (lets say a YouTube video)  loads for the user does it count towards the 100GB Bandwidth per month?</p>
<p>I want to swap out most of my images for embeds from my google photo storage to save on my 1GB max website size and i was wondering if doing this will also save on my bandwidth cap.<br>
Thanks</p>

**Comment by tajmone** [comment, marked as answer]
<p>If those videos are handled on the client side (the browser) than they are not transiting through the GitHub servers, so they won’t affect your account bandwidth.</p>

**Comment by Senseal** [comment]
<p>Thanks Bro<br>
(some random Jargen to make github happy because of its crap 20 character limit)</p>
